front/src/app/scenes/address/address.component.html

```diff
@@ -8,14 +8,22 @@
             <dl>
               <dt>Address hash:</dt>
               <dd>
-                {{addr.address}}
+                <span class="text-monospace">{{addr.address}}</span>
                 <ng-container *ngIf="addr.signerDetails">
                   (<a [href]="addr.signerDetails.url" target="_blank">{{addr.signerDetails.name}}</a>)
                 </ng-container>
               </dd>
               <dt>Balance:</dt>
               <dd>{{addr.balance_wei | weiToGO | bigNumber}}</dd>
               <ng-container *ngIf="addr.contract">
+                <ng-container *ngIf="addr.target">
+                  <dt>Target:</dt>
```

Comment:
There will be an `Upgradeable` tag nearby, but is there a better way to communicate what this is? Would a little `?` icon tooltip be helpful? (e.g. `Upgradeable contracts execute the code which they target`)

Reply:
Let's merge and then update this in a separate PR